src/mocks/data.ts

```diff
@@ -203,8 +203,8 @@ export const mockEvent: IsaacEventPageDTO = {
     },
   ],
   tags: ["virtual", "student", "booster"],
-  date: 1698850800000 as unknown as Date,
-  bookingDeadline: 1698840000000 as unknown as Date,
+  date: 253399842000000 as unknown as Date,
+  bookingDeadline: 253399755600000 as unknown as Date,
```

Comment:
Since this is not the first time that we have problems with tests and dates, wouldn't it be better to create a helper function that creates a date in the future (e.g. current date + 2 months)? That way the intention is clearer, and we can reuse that for any dates that are meant to be in the future.

Reply:
Yes exactly that's my intention. I didn't see much value to sorting that until we fix the `Date` typing issue though, and I can do it all in one.